pkgs/top-level/python-packages.nix

```diff
@@ -7552,6 +7552,10 @@ self: super: with self; {
 
   litemapy = callPackage ../development/python-modules/litemapy { };
 
+  litestar = toPythonModule (pkgs.litestar.override {
+    python3Packages = pythonPackages;
```

Comment:
Shouldn'ts litestar be moved under `pkgs/development/python-modules` instead and this be added to `python-packages.nix`? I'm pretty sure this is the convention for python packages.